diff --git a/node_modules/@virgilsecurity/e3kit-native/dist/e3kit-native.cjs.js b/node_modules/@virgilsecurity/e3kit-native/dist/e3kit-native.cjs.js
index 9b15499..34dd2e0 100644
--- a/node_modules/@virgilsecurity/e3kit-native/dist/e3kit-native.cjs.js
+++ b/node_modules/@virgilsecurity/e3kit-native/dist/e3kit-native.cjs.js
@@ -34,7 +34,8 @@
 
 var reactNativeVirgilCrypto = require('react-native-virgil-crypto');
 var reactNativeKeychain = require('react-native-keychain');
-var require$$0$4 = require('react-native');
+var require$$0$4 = require('@react-native-async-storage/async-storage');
+//var require$$0$4 = require('react-native');
 
 var global$1 = (typeof global !== "undefined" ? global :
             typeof self !== "undefined" ? self :
diff --git a/node_modules/@virgilsecurity/e3kit-native/dist/types/EThree.d.ts b/node_modules/@virgilsecurity/e3kit-native/dist/types/EThree.d.ts
index 89b89fe..b57e306 100644
--- a/node_modules/@virgilsecurity/e3kit-native/dist/types/EThree.d.ts
+++ b/node_modules/@virgilsecurity/e3kit-native/dist/types/EThree.d.ts
@@ -3,10 +3,10 @@ import { AbstractEThree } from '@virgilsecurity/e3kit-base';
 import { Data, EThreeCtorOptions, EThreeInitializeOptions } from './types';
 import './asyncstoragedown-clear-polyfill';
 export interface EThreeNativeInitializeOptions extends EThreeInitializeOptions {
-    AsyncStorage: import('react-native').AsyncStorageStatic;
+    AsyncStorage: import('@react-native-async-storage/async-storage').AsyncStorageStatic;
 }
 export interface EThreeNativeCtorOptions extends EThreeCtorOptions {
-    AsyncStorage: import('react-native').AsyncStorageStatic;
+    AsyncStorage: import('@react-native-async-storage/async-storage').AsyncStorageStatic;
 }
 export declare class EThree extends AbstractEThree {
     /**
