diff --git a/node_modules/pmtiles/index.ts b/node_modules/pmtiles/index.ts
index 3d5a43e..e5223c1 100644
--- a/node_modules/pmtiles/index.ts
+++ b/node_modules/pmtiles/index.ts
@@ -414,15 +414,15 @@ export class FetchSource implements Source {
       throw Error(`Bad response code: ${resp.status}`);
     }
 
-    // some well-behaved backends, e.g. DigitalOcean CDN, respond with 200 instead of 206
-    // but we also need to detect no support for Byte Serving which is returning the whole file
-    const contentLength = resp.headers.get("Content-Length");
-    if (resp.status === 200 && (!contentLength || +contentLength > length)) {
-      if (controller) controller.abort();
-      throw Error(
-        "Server returned no content-length header or content-length exceeding request. Check that your storage backend supports HTTP Byte Serving."
-      );
-    }
+    // // some well-behaved backends, e.g. DigitalOcean CDN, respond with 200 instead of 206
+    // // but we also need to detect no support for Byte Serving which is returning the whole file
+    // const contentLength = resp.headers.get("Content-Length");
+    // if (resp.status === 200 && (!contentLength || +contentLength > length)) {
+    //   if (controller) controller.abort();
+    //   throw Error(
+    //     "Server returned no content-length header or content-length exceeding request. Check that your storage backend supports HTTP Byte Serving."
+    //   );
+    // }
 
     const a = await resp.arrayBuffer();
     return {
