diff --git a/node_modules/react-native-modal-selector/index.js b/node_modules/react-native-modal-selector/index.js
index 9b8adf6..5f326f9 100644
--- a/node_modules/react-native-modal-selector/index.js
+++ b/node_modules/react-native-modal-selector/index.js
@@ -9,7 +9,7 @@ import {
     Text,
     FlatList,
     ScrollView,
-    TouchableOpacity,
+    Pressable,
     TouchableWithoutFeedback,
 } from 'react-native';
 
@@ -240,7 +240,7 @@ export default class ModalSelector extends React.Component {
         );
 
         return (
-            <TouchableOpacity
+            <Pressable
               key={this.props.keyExtractor(option)}
               testID={option.testID || this.props.optionsTestIDPrefix + '-' + optionLabel}
               onPress={() => this.onChange(option)}
@@ -253,7 +253,7 @@ export default class ModalSelector extends React.Component {
                 <View style={[styles.optionStyle, this.props.optionStyle, isLastItem && {borderBottomWidth: 0}]}>
                   {component}
                 </View>
-            </TouchableOpacity>);
+            </Pressable>);
     }
 
     renderFlatlistOption = ({ item, index, separators }) => {
@@ -340,11 +340,11 @@ export default class ModalSelector extends React.Component {
                         }
                     </View>
                     <View style={[styles.cancelContainer, cancelContainerStyle]}>
-                        <TouchableOpacity onPress={this.close} activeOpacity={touchableActiveOpacity} accessible={cancelButtonAccessible} accessibilityLabel={cancelButtonAccessibilityLabel}>
+                        <Pressable onPress={this.close} activeOpacity={touchableActiveOpacity} accessible={cancelButtonAccessible} accessibilityLabel={cancelButtonAccessibilityLabel}>
                             <View style={[styles.cancelStyle, cancelStyle]}>
                                 <Text style={[styles.cancelTextStyle,cancelTextStyle]} {...this.props.cancelTextPassThruProps}>{cancelText}</Text>
                             </View>
-                        </TouchableOpacity>
+                        </Pressable>
                     </View>
                 </View>
             </Overlay>
@@ -387,7 +387,7 @@ export default class ModalSelector extends React.Component {
                 {this.props.customSelector ?
                     this.props.customSelector
                     :
-                    <TouchableOpacity
+                    <Pressable
                         hitSlop={this.props.modalOpenerHitSlop}
                         activeOpacity={this.props.touchableActiveOpacity}
                         style={this.props.touchableStyle}
@@ -399,7 +399,7 @@ export default class ModalSelector extends React.Component {
                         <View style={this.props.childrenContainerStyle} pointerEvents="none">
                             {this.renderChildren()}
                         </View>
-                    </TouchableOpacity>
+                    </Pressable>
                 }
             </View>
         );
