diff --git a/node_modules/react-native-zip-archive/android/build.gradle b/node_modules/react-native-zip-archive/android/build.gradle
index 5f71141..c4bdb63 100644
--- a/node_modules/react-native-zip-archive/android/build.gradle
+++ b/node_modules/react-native-zip-archive/android/build.gradle
@@ -37,5 +37,6 @@ repositories {
 
 dependencies {
     implementation "com.facebook.react:react-native:+"
-    implementation group: 'net.lingala.zip4j', name: 'zip4j', version: '2.+'
+    implementation "net.lingala.zip4j:zip4j:2.11.5"
+    
 }
